### ISSUE TITLE
error: TypeError: Wallet.createpayment is not a function

### ISSUE THREAD

**Issue by TheNeanderthal** [blockchain/service-my-wallet-v3#84]
service-my-wallet-v3 was pulled from github, 2 days ago. 

Service: v0.19.0
Node: v4.4.0

![node](https://cloud.githubusercontent.com/assets/17935162/13890222/928172b6-ed07-11e5-882e-e0e03d5417fa.png)

It seems to be biased, it was working perfectly for the first day, now the 2nd I get this error constantly. Looks like it has a problem just sending the money?

Here is the code it's failing on, grabbed from PHP error log,

> [Fri Mar 18 15:30:09.886855 2016] [:error] [pid 19834] [client 127.0.0.1:54764] PHP Warning:  file_get_contents(http://localhost:3030/merchant/$GUID/payment?password=$password1&second_password=$password2&to=$sendaddress&amount=36916715&api_code=$apicode): failed to open stream: HTTP request failed! HTTP/1.1 500 Internal Server Error\r\n in /var/www/html/functions.php on line 25

I redacted some information by replacing with $GUID, so on

My sendpayment function:

```
function SendPayment($guid,$walletpassword,$address,$amount,$apicode){
		$amount = ($amount *  100000000);
		$amount = $amount - 20000;
		file_get_contents("http://localhost:3030/merchant/$guid/payment?password=$walletpassword&second_password=$walletpassword&to=$address&amount=$amount&api_code=$apicode");
		
		
		return 1;
	}
```

Just trying to supply as much information as possible

Just restarted blockchain-wallet-service and seems to be working, but nonetheless this is still an issue.

**Comment by jtormey** [comment]
Hey, thanks for reaching out, sorry you're having this issue. It looks like this bug was introduced in v0.19, and only happens under specific conditions (I've been having trouble reproducing it, and could explain why you didn't see it at first).

With the info you gave me I was able to create a patch and released it as v0.19.2. Can you try upgrading to see if you still get this error? Thanks again.

**Comment by TheNeanderthal** [comment]
Hey! Sorry for the extremely late reply, have been busy. I pulled the new wallet service from github a couple days after you said you fixed and so far it has been running flawlessly.
